src/initialization.js

```diff
@@ -0,0 +1,286 @@
+const Apify = require('apify');
+const _ = require('lodash');
+const HeaderGenerator = require('header-generator');
+const { TYPES, LABELS } = require('./constants');
+
+const fns = require('./functions');
```

Comment:
Maybe deconstruct the functions here?

Reply:
The whole `fns` is passed as a helper to `extendFunction` and I'm not sure how exactly it is intended to work. I'd just leave it this way to be safe.